experimental/c-cpp/manager.py

```diff
@@ -923,6 +1011,37 @@ def log_fuzzer_source(full_fuzzer_source: str):
   logger.info(harness_source_out)
 
 
+def get_tests_converted_to_harnesses(build_results, language, test_dir,
+                                     fuzzer_build_cmd, all_header_files,
+                                     all_test_scripts, github_url):
+  """Converts a list of test files into fuzzing harnesses."""
+  return_list = []
+
+  _, _, fuzzer_target_file, _ = get_language_defaults(language)
+
+  for test in all_test_scripts:
+
+    fuzz_converter = FuzzerGenHeuristicTestConverter({}, all_header_files,
+                                                     test_dir)
+    fuzz_converter.github_url = github_url
+    fuzzer_intrinsics = fuzz_converter.get_fuzzer_test_intrinsics(test)
+
+    # Generate a build script for compiling the fuzzer with ASAN.
+    final_asan_build_script = build_results[test_dir].build_script + '\n'
+    fuzzer_out = '/src/generated-fuzzer'
+    fuzz_cmd = ' '.join(fuzzer_build_cmd) + ' '
```

Comment:
May I ask why the trailing space is necessary, given `final_asan_build_script += f'{fuzz_cmd} {fuzz_includes} ...'`?

Reply:
good catch, there's no need.